responsive-grid-layout/tugba/styles.css

```diff
@@ -0,0 +1,55 @@
+*{
+   margin: 0;
+   padding: 0; 
+   border: 1px blue solid;
+}
+body{
+    background-color: pink;
+    margin: 0 ;
+    font-size: 26px;
+
+}
+
+.container {
+    height: 100vh;
+    display: grid;
+    grid-template-columns: 1fr 4fr 1fr;
+    grid-template-rows: 80px 1fr 80px;
+    grid-template-areas:  
+    "header header header"
+    "sidebar-left content sidebar-right"
+    "sidebar-left footer footer"
+}
+
+header {
+    display: grid;
+    background-color: aqua;  
+    grid-template-columns: 1 / span 5;
```

Comment:
`grid-template-columns: 1 / span 5;` is one of the ways to make the layout look as we need it to. However, using 'grid-area: header;'is a cleaner way to do that. You don't need to use both of them at the same time.